DEVELOPMENT.md

```diff
@@ -125,36 +249,4 @@ try:
     my_function("key", "value")
 except ValueError as e:
     # Error handling...
-```
-
-## About the data model
```

Comment:
Did we want to remove this data model section? I noticed that the super verbose section was moved but not this one.

Reply:
Yeah, the data model being in here was a copy/paste mistake. The pydantic models are only in the openjd-model package, not this one.